main/java/com/epam/dao/EntityDao.java

```diff
@@ -0,0 +1,77 @@
+package com.epam.dao;
+
+import com.epam.entity.Currency;
+import com.epam.entity.UserAccount;
+import com.epam.exceptions.IncorrectAccountException;
+import com.epam.exceptions.IncorrectCurrencyException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.math.BigDecimal;
+import java.util.Properties;
+
+/**
+ * Created by Almas_Doskozhin
+ * on 2/22/2016.
+ */
+public class EntityDao {
```

Comment:
To be honest, I don't like your DAO.
* You throw low level exception and it means that service layer must be aware about low-level exceptions
* You don't close streams
* you have copy-pasted code